docs/examples/widgets/selection_list_selected.py

```diff
@@ -0,0 +1,40 @@
+from textual import on
+from textual.app import App, ComposeResult
+from textual.containers import Horizontal
+from textual.events import Mount
+from textual.widgets import Footer, Header, Pretty, SelectionList
+from textual.widgets.selection_list import Selection
+
+
+class SelectionListApp(App[None]):
+    CSS_PATH = "selection_list_selected.css"
+
+    def compose(self) -> ComposeResult:
+        yield Header()
+        with Horizontal():
+            yield SelectionList[str](  # (1)!
+                Selection("Falken's Maze", "secret_back_door", True),
+                Selection("Black Jack", "black_jack"),
```

Comment:
Isn't the game "Blackjack"? 😆

Reply:
No!

<img width="569" alt="Screenshot 2023-05-25 at 12 57 41" src="https://github.com/Textualize/textual/assets/28237/e4e7d08c-aadb-4fb6-870e-cdde14534e7d">